### PR TITLE
Add banner instantsearch-react

### PR DESCRIPTION
**Pull request by LukyVj**
Banner to instantsearch-react added on landing/doc

### CODE REVIEW

File: docs/_plugins/toc_generator.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ def toc_generate(html)
         if level > current_level
           current_level.upto(level - 1) do
             html << '<ul>'
+            html << '<li><a href=\"#booya\">react-instantsearch</a></li>'
```
Comment (vvo) [inline docs/_plugins/toc_generator.rb]: booya?

File: docs/react/assets/data/community-projects.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[
```
Comment (vvo) [inline docs/react/assets/data/community-projects.json]: You need to remove completely the docs/ folder from your PR

File: docs/_plugins/toc_generator.rb (hunk 1 above)
Comment (LukyVj) [inline docs/_plugins/toc_generator.rb]: Oops

File: docs/_layouts/documentation.html [hunk 3]
```diff
@@ -5,6 +5,8 @@
 <div class="container-fluid main-documentation">
   <div class="row">
     <div class="col-lg-2 col-md-3 hidden-xs hidden-sm widget-nav">
+      <a href="/react/" class="banner-button">Try React InstantSearch <img src="http://res.cloudinary.com/hilnmyskv/image/upload/v1481540222/react-favicon_y43wx4.svg" alt="react-instantsearch icon"></a>
```
Comment (vvo) [inline docs/_layouts/documentation.html]: Put an absolute link, will not work otherwise

File: docs/_config.yml [hunk 4]
```diff
@@ -14,6 +14,8 @@ navigation:
    url: /examples/
  - text: About
    url: /about/
+ - text: react-instantsearch
+   url: /react/
```
Comment (vvo) [inline docs/_config.yml]: put absolute link, check it works

File: docs/_config.yml [hunk 5]
```diff
@@ -14,6 +14,8 @@ navigation:
    url: /examples/
  - text: About
    url: /about/
+ - text: react-instantsearch
```
Comment (vvo) [inline docs/_config.yml]: Text should be React InstantSearch

File: docs/_plugins/toc_generator.rb [hunk 6]
```diff
@@ -15,6 +15,7 @@ def toc_generate(html)
         if level > current_level
           current_level.upto(level - 1) do
             html << '<ul>'
+            html << '<li><a href=\"https://community.algolia.com/instantsearch.js/react/\">react-instantsearch</a></li>'
```
Comment (vvo) [inline docs/_plugins/toc_generator.rb]: Text should be React InstantSearch

File: docs/index.haml [hunk 7]
```diff
@@ -1,6 +1,14 @@
 ---
 layout: default
 ---
+.marketing-banner
+  %a{:href => "#closeBanner", :class => "banner-close"} &#10005;
+  .banner__description
+    %a{:href => "https://community.algolia.com/instantsearch.js/react/"}
+      %p <span>Looking for a React version of instantsearch.js?</span> Have a look at React InstantSearch!
+
+      %img{:src => "http://res.cloudinary.com/hilnmyskv/image/upload/v1481540222/react-favicon_y43wx4.svg", :alt => "React-instantsearch icon"}/
```
Comment (vvo) [inline docs/index.haml]: React InstantSearch

File: docs/_config.yml [hunk 8]
```diff
@@ -14,6 +14,8 @@ navigation:
    url: /examples/
  - text: About
    url: /about/
+ - text: React InstatSearch
```
Comment (vvo) [inline docs/_config.yml]: Typo

**Pull request merged**